### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/client-platforms/aurelia/01-login.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  githubUrl: 'https://github.com/auth0/auth0-aurelia',
-  pkgOrg: 'auth0',
-  pkgRepo: 'auth0-aurelia',
+  pkgRepo: 'auth0-aurelia-samples',
```
Comment (mpaktiti) [inline articles/client-platforms/aurelia/01-login.md:14]: @mehreencs87 Hi, the `pkgOrg` is missing, can you add it please?

**Comment by mpaktiti** [comment]
Hi @mehreencs87 can you please add the missing `pkgOrg` to this PR? Thank you.

**Comment by mehreencs87** [comment]
@mpaktiti updated,

File: articles/client-platforms/aurelia/01-login.md [hunk 2]
```diff
@@ -11,15 +11,16 @@ This tutorial and seed project have been tested with the following:
...
+  githubUrl:'https://github.com/auth0-samples/auth0-aurelia-samples/tree/master/00-Starter-Seed',
+  pkgOrg:'auth0-samples',
   pkgBranch: 'master',
-  pkgPath: null,
+  pkgPath: 00-Starter-Seed,
```
Comment (mpaktiti) [inline articles/client-platforms/aurelia/01-login.md]: @mehreencs87 Can you place `00-Starter-Seed` in single quotes please?

**Comment by mehreencs87** [comment]
@mpaktiti done.

**Pull request merged**